### PR TITLE
Article Loading Experience

### PR DESCRIPTION
**Pull request by hueitan**

### CODE REVIEW

File: src/components/Loading.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      history.back()
+    }
+  }
+
+  useSoftkey('Loading', {
+    right: i18n.i18n('softkey-close'),
+    onKeyRight: onClose
+  })
+
+  return <div class='loading-planet'>
```
Comment (stephanebisson) [inline src/components/Loading.js]: There should be parenthesis around the JSX here.

File: src/components/Loading.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { h } from 'preact'
+import { useSoftkey, useI18n } from 'hooks'
+export const Loading = ({ close, message }) => {
+  const i18n = useI18n()
+
+  const onClose = () => {
+    if (close) {
```
Comment (stephanebisson) [inline src/components/Loading.js]: Is the `close` property used anywhere?

Comment (hueitan) [reply src/components/Loading.js]: I'm thinking of `<Loading/>` can be used either as a single page or popup content, but yes we are only using this in article so far, so I will remove this line until we need it.

File: src/components/Loading.js (hunk 1 above)
Comment (hueitan) [reply src/components/Loading.js]: I'm not sure what it means here, but this returns a single <div/>, that's fine wihout having the parenthesis here, is there any code styling concern here?

Comment (stephanebisson) [reply src/components/Loading.js]: This is for consistency with the other components. Put `(` right after `return` and start the JSX on the next line. This way the markup is indented correctly.

**Pull request merged**